uniswapv3/wrapper/src/utils/tokenUtils.ts

```diff
@@ -24,7 +24,7 @@ export function _getNative(chainId: ChainId): Token {
   if (chainId < ChainId.MAINNET || chainId >= ChainId._MAX_) {
     throw new Error("Unknown chain ID");
   }
-  if (chainId == ChainId.POLYGON || chainId == ChainId.POLYGON_MUMBAI) {
+  if (chainId >= ChainId.POLYGON) {
```

Comment:
Can you explain this change further? It seems like too broad of an assumption to make... maybe a range would be better?

Reply:
I made the change because ChainId.POLYGON and ChainId.POLYGON_MUMBAI are the last values in the enum, which makes it possible to evaluate the condition with only one operation

Reply:
I changed it back